argo-cd-apps/base/all-clusters/infra-deployments/external-secrets-operator/external-secrets-operator.yaml

```diff
@@ -20,6 +20,9 @@ spec:
                   values.clusterDir: stone-stage-p01
                 - nameNormalized: stone-stg-rh01
                   values.clusterDir: stone-stg-rh01
+                - nameNormalized: pentest-p01
+                  values.environment: production
```

Comment:
We need this here, we patch all application set  environment value in prod overlay already.

Reply:
Done.